app/views/directives/osc-routing.html

```diff
@@ -160,19 +160,26 @@ <h3>Alternate Services</h3>
           </div>
         </div>
       </div>
-
+      
       <div ng-repeat="alternate in route.alternateServices" class="form-group">
         <osc-routing-service model="alternate"
-                             service-options="alternateServiceOptions"
-                             all-services="servicesByName"
-                             is-alternate="true"
-                             show-weight="route.alternateServices.length > 1 || controls.hideSlider">
+          service-options="alternateServiceOptions"
+          all-services="servicesByName"
+          is-alternate="true"
+          show-weight="route.alternateServices.length > 1 || controls.hideSlider">
         </osc-routing-service>
-        <a href="" ng-click="route.alternateServices.splice($index, 1)">Remove Service</a>
-        <span ng-if="$last && route.alternateServices.length < alternateServiceOptions.length">
-          <span class="action-divider">|</span>
-          <a href="" ng-click="addAlternateService()">Add Another Service</a>
-        </span>
+        <div class="row">
+          <div class="col-sm-6">
+            <a href="" ng-click="route.alternateServices.splice($index, 1)">Remove Service</a>
+            <span ng-if="$last && route.alternateServices.length < alternateServiceOptions.length">
+              <span class="action-divider">|</span>
+              <a href="" ng-click="addAlternateService()">Add Another Service</a>
+            </span>
+          </div>
+          <div ng-if="route.alternateServices.length === 1 && controls.hideSlider" class="col-sm-6">
+            <a href="" ng-click="controls.hideSlider = false">Edit Weights Using Percentage Slider</a>
```

Comment:
We're trying to switch to `button` elements styled as links in new code for better accessibility. (I realize you've followed what was there.)

```html
<button type="button" ng-click="controls.hideSidebar = false" class="btn btn-link">Edit Weights Using Percentage Slider</button>
```

Please check that it doesn't break using enter to submit the form, however. If it does, we might just leave it an `<a>` for now.

Reply:
I was about to ask about this switch, glad I read your comment 😄